ksql-rest-app/src/test/java/io/confluent/ksql/rest/server/computation/CommandRunnerTest.java

```diff
@@ -59,16 +66,25 @@ private CommandRunner getCommanRunner(StatementExecutor statementExecutor) {
     return commandRunner;
   }
 
+  private StatementExecutor getStatementExecutor() {
+    Map<String, Object> props = new HashMap<>();
+    props.put("application.id", "ksqlStatementExecutorTest");
+    props.put("bootstrap.servers", CLUSTER.bootstrapServers());
 
-  @Test
-  public void testThread() throws InterruptedException {
     MockKsqkEngine mockKsqkEngine = new MockKsqkEngine(
-        TestUtils.getMockKsqlConfig(), new MockKafkaTopicClient());
+        new KsqlConfig(props), new MockKafkaTopicClient());
+
     StatementParser statementParser = new StatementParser(mockKsqkEngine);
-    StatementExecutor statementExecutor = new StatementExecutor(mockKsqkEngine, statementParser);
+
+    return new StatementExecutor(mockKsqkEngine, statementParser);
+  }
+
+  @Test
+  public void testThread() throws InterruptedException {
+    StatementExecutor statementExecutor = getStatementExecutor();
     CommandRunner commandRunner = getCommanRunner(statementExecutor);
     new Thread(commandRunner).start();
-    Thread.sleep(50000);
+    Thread.sleep(2000);
```

Comment:
We should be avoiding `Thread.sleep(...)` in tests. With the code the way it currently is, you could pass a mock/stub version of the `StatementExectuor` to the `CommandRunner` and use a latch or counter to determine when it has received the expected commands.

Reply:
Refactored this test along with the CommandRunner test so we won't need to test the thread.